apps/ui/src/pages/review/index.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2025 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { Button, Flex, Layout, theme, Typography } from 'antd';
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { API_PATH_LOGIN } from '@/api/paths';
+import {
+	clearExtraSessionInformation,
+	getExtraSessionInformation,
+	setExtraSessionInformation,
+} from '@/global/localStorage';
+import { pcglColours } from '@/providers/ThemeProvider';
+import { useUserContext } from '@/providers/UserProvider';
+import { useMatch, useNavigate } from 'react-router';
+
+const { useToken } = theme;
+const { Content } = Layout;
+const { Text, Title } = Typography;
+
+const InstitutionalRepLogin = () => {
+	const { token } = useToken();
+	const { t: translate } = useTranslation();
+	const navigation = useNavigate();
+	const { isLoggedIn, role: loggedInRole } = useUserContext();
+	const match = useMatch('review/:applicationId');
+
+	const landingPageOuter: React.CSSProperties = {
+		backgroundColor: pcglColours.white,
+		display: 'flex',
+	};
+
+	const landingPageInner: React.CSSProperties = {
+		backgroundColor: pcglColours.tertiary,
+		margin: token.marginLG,
+		padding: token.marginLG,
+	};
+
+	const landingPageDescription: React.CSSProperties = {
+		maxWidth: '35rem',
+	};
+
+	useEffect(() => {
+		if (isLoggedIn && loggedInRole !== 'INSTITUTIONAL_REP') {
+			navigation('/login/redirect', { replace: true });
+		}
```

Comment:
Repeat of previous comment... Can't remember what we decided for where this redirect logic should live, but it feels like it should be in the router component not in a page's use effect.